juju/juju
# Add new remove-saas CLI

## Description of change

juju remove-application is no longer used to remove a SAAS application.
There's now a remove-saas command, aliased to remove-consumed-application

## QA steps

consume an offer, see that it is in status under SAAS
`juju remove-saas <saas name>`
run status and check that it's gone

## Documentation changes

We'll need to document the new remove-saas command, and adjust doc for remove-application
